metrics/ignores/platform-all.json

```diff
@@ -85,6 +85,7 @@
     "render-tests/fill-pattern/update-feature-state": "https://github.com/mapbox/mapbox-gl-native/issues/15895",
     "render-tests/geojson/inline-linestring-fill": "current behavior is arbitrary",
     "render-tests/icon-text-fit/text-variable-anchor-overlap": "https://github.com/mapbox/mapbox-gl-native/issues/15809",
+    "render-tests/icon-text-fit/textFit-grid-long": "Needs to be investigated and fixed.",
```

Comment:
Maybe we can only ignore it on the platforms where it is failing?

For me it is running fine locally on Android.

If you would prefer to merge this and look into this later, let's create an issue so we don't forget.

Reply:
Oh, it's because that I'm not sure if the online Android render test also failed with this render test, (it also running fine locally on my side), but it seems not, so I'll move this to the `platform-iOS`.

Reply:
Well, looks like it failed on the Pixel 7 Pro on CI.

Might be device-dependent. Maybe best to disable it after all so we can merge this.

Reply:
Yeah, but I don't know why is it failed since we couldn't obtain the result file now, I think your PR will be helpful.